mne/utils.py

```diff
@@ -252,6 +252,43 @@ def sum_squared(X):
     return np.dot(X_flat, X_flat)
 
 
+def warn(message, category=RuntimeWarning):
+    """Emit a warning with trace outside the mne namespace
+
```

Comment:
Since future developers will also need this function, maybe we can explain a bit more here:

Warnings can be generated deep within nested function calls. In order to provide a more helpful point of origin for the warning, this function traverses the stack until it reaches the line in the user script responsible for the error.